main.rb

```diff
@@ -29,17 +30,61 @@ def choose_number(choice, app)
 end
 
 def main
-  app = App.new
+  # load the data and then parse it to the app
 
+  books = load_book_data
+  persons = load_person_data
```

Comment:
- [ ] When I try to run the app, I have an error raised (The one on the image below).
This error happens because you used two different names for the same method `load_person_data` and `load_persons_data`, Please change one of them so it can match the other to fix this error because you changed the method name while trying to add the requested changes.

![image](https://github.com/brighton0102/School-Library-App/assets/68796232/8ea1a623-3817-46c8-83ac-e41219083858)